src/Alert/Alert.js

```diff
@@ -104,6 +104,23 @@ export const Alert = ({ classes = {}, children, ...props }) => {
         );
         break;
 
+      case 'errorSummary':
+        levers.wrapper =
+          'su-bg-digital-red su-bg-opacity-20 su-text-digital-red su-link-digital-red';
+        levers.body = redText;
+        levers.dismiss = 'red';
+        levers.dismissIcon = 'x';
+        defaultIcon = (
+          <Icon
+            icon="exclamation-circle"
+            type="solid"
+            aria-hidden="true"
+            className={classes.icon}
+            {...iconProps}
+          />
+        );
+        break;
```

Comment:
Looks like other than the icon on the left, the Form Error alerts will use a different dismiss button too (without the "dismiss" word) - there's an example story we could use.

I'd also add a variable to change out the alignment of the left icon and have it top align with the alert content if it's in a summary box 🤔 

The hover:su-link-underline/focus:su-link-underline etc classes were problematic for the JIT mode, so Shea has removed them in the latest version of Decanter v7. the hover:su-link-somecolor ones are also problematic 😬  You've probably noticed that they don't work very well for the links at the moment. 

I think it would be nice for us to abstract a link list component in Decanter React, and then we can use that component here in the Error Summary (but that can be in another ticket). That way we can use children:hocus: to add underline or color hocus styles to the links within a list.